.github/infrastructure/docker-compose-mysql.yml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
   db:
-    image: mysql
+    image: mysql:8
```

Comment:
Better to pin a specific version. We have learnt that lesson with Redis earlier today...